### ISSUE TITLE
Zulip doesn't remember login credentials

### ISSUE THREAD

**Issue by jjst** [zulip/zulip-electron#10]
I have to re-enter my login credentials (email/password) each time I launch the app.

Platform: Ubuntu 16.04

**Comment by akashnimare** [comment]
@jist I have yet to implement this.

**Comment by akashnimare** [comment]
@jjst I can't reproduce this. I am running it on Ubuntu 14.04. I entered my login credentials only once when I first launched app. After that it will automatically saved credentials in local storage (same as web version). Let me know if you are still having this problem.

**Comment by timabbott** [comment]
@akashnimare could this have been during the brief period when #12 was present (i.e. it was pointing at a different server by default)?  That would certainly have been experienced as, in part, needing to re-enter login credentials.

**Comment by akashnimare** [comment]
I don't think this is the reason behind this. The behaviour of app is same as loading that targetUrl in chrome.

@jjst Follow this steps and let me know what happens -
> Set targetUrl to https://github.com
> Start app and login to github using email/pass (in app window)
> Quit app
> Start app again and see if you are logged in or not.

**Comment by akashnimare** [comment]
@jjst Is this resolved?

**Comment by jjst** [comment]
Reproduced the issue on 33fb06c. I followed your instructions and when setting the targetUrl to github's, I can login, relaunch the app and I'm still logged in. But when I revert back to "zulip.com/login", I get logged off when restarting the app.

**Comment by akashnimare** [comment]
Can you check this with latest code? It shouldn't behave like this.

**Comment by akashnimare** [comment]
I am not having this problem when using domain https://zulip.tabbott.net (Using email/pass).
I don't have login credentials for https://zulip.com/login so I can't reproduce this either.

**Comment by akashnimare** [comment]
I can't reproduced this so closing until more information is given.

**Comment by jjst** [comment]
@akashnimare Apologies for the lack of feedback, my bad :-(

I'm trying to see if it still happens on the latest code, but I'm now seeing a blank window when launching the app. I'll open a separate ticket.

**Comment by akashnimare** [comment]
Pull the latest code and remove domain.json and then npm start.

**Comment by jjst** [comment]
Thanks, I actually just did it the bulldozer way and did a fresh git clone and it works. I can also confirm that I'm not experiencing the login credentials anymore, I restarted the app multiple times and am still logged in!

**Comment by akashnimare** [comment]
Awesome :boom: Finally :smiley:
